backend/models/base.go

```diff
@@ -2,13 +2,34 @@ package models
 
 import (
 	"time"
+
+	"github.com/google/uuid"
 	"gorm.io/gorm"
 )
 
+type ModelID struct {
```

Comment:
I think this is fine but it may simplify things if we can use Postgresqls internal UUID generation. I don't know how that would look with GORM or Golang though.